ci-operator/step-registry/openshift/e2e/azure/csi/file/cross-subscription/openshift-e2e-azure-csi-file-cross-subscription-workflow.yaml

```
@@ -0,0 +1,40 @@
+workflow:
+  as: openshift-e2e-azure-csi-file-cross-subscription
+  steps:
+    pre:
+    - ref: azure-provision-resourcegroup
+    - ref: azure-provision-vnet
+    - chain: azure-provision-bastionhost
+    - chain: ipi-conf-azure
+    - ref: ipi-conf-proxy
+    - ref: ipi-conf-azure-provisionednetwork
+    - ref: azure-provision-resourcegroup-cross-subscription
+    - ref: storage-conf-storageclass-pre-set-default-azure-file-cross-subscription
+    - chain: ipi-install
+    - ref: storage-conf-csi-azure-file
+    - ref: storage-conf-wait-for-csi-driver
+    - ref: storage-obj-save
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - ref: storage-obj-check
+    - chain: gather-core-dump
+    - ref: gather-proxy
+    - chain: ipi-azure-post
+    - ref: cucushift-installer-check-azure-destroy-dns
+    - ref: azure-deprovision-resourcegroup
+    env:
+      # for storage-conf-wait-for-csi-driver
+      CLUSTERCSIDRIVER: file.csi.azure.com
+      TRUECONDITIONS: AzureFileDriverControllerServiceControllerAvailable AzureFileDriverNodeServiceControllerAvailable
+      TEST_CSI_DRIVER_MANIFEST: manifest-azure-file.yaml
+      TEST_OCP_CSI_DRIVER_MANIFEST: ocp-manifest-azure-file.yaml
+      TEST_SUITE: openshift/csi
+      # skip cases due to https://issues.redhat.com/browse/OCPBUGS-54897
+      TEST_SKIPS: >-
+        \[Feature:VolumeSnapshotDataSource\]
```

```suggestion
      # TODO: skip cases due to https://issues.redhat.com/browse/OCPBUGS-54897
      # remove the skip after the bug fix merge and backport
      TEST_SKIPS: >-
        \[Feature:VolumeSnapshotDataSource\]
```
Better to add a `TODO` for this.